.travis.yml

```diff
@@ -1,38 +1,109 @@
-dist: trusty
-
+dist: bionic
 language: cpp
 
-matrix:
+branches:
+  only:
+    - master
+    - development
+
+compiler:
```

Comment:
I think this section cause the tests to run twice. I suspect you can just remove it and it will be fine.

Reply:
Yes, since the original repository doesn't have a `development` branch, it's not needed anymore. I'm thinking of removing it in my fork as well.

Reply:
I had in might the "compiler" section which cause double run for each compiler I think. (Not sure)

Reply:
Yes, you're completely right! I removed the `compiler` listing and cleared up the `jobs` definitions. Now it runs only 1 build per compiler/job and identifies the compiler correctly.